code/mob/living/carbon/human.dm

```diff
@@ -1888,6 +1888,7 @@
 
 /mob/living/carbon/human/updateTwoHanded(var/obj/item/I, var/twoHanded = 1)
 	if(!(I in src) || (src.l_hand != I && src.r_hand != I)) return 0
+	if (!(src.has_hand(1) && src.has_hand(0))) return 0 //gotta have two hands to two-hand
```

Comment:
```suggestion
	if (!(src.has_hand(1) && src.has_hand(0))) return FALSE //gotta have two hands to two-hand
```
bools good 0/1 bad

Reply:
Well the rest of that function uses 0/1, so we may as well fix that. Let me just check that doesn't break anything

Reply:
actually TRUE and FALSE are just macros for 0 and 1 anyway, so this is always safe, right?

Reply:
Yep, use TRUE/FALSE whenever you can, replacing other 0/1s with FALSE/TRUE is a net positive (provided it will only ever be those two, for the sake of clarity)